src/index.html

```diff
@@ -15,8 +15,108 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__logo">
+          <a href="#">
+            <img
+              src="images/logo.png"
+              alt="moyo logo"
+            />
+          </a>
+        </div>
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a
+                href="#"
+                class="nav__link is-active"
```

Comment:
If you are using BEM, here is mistake. You can't use `nav__link` element if you don't have `nav` block

Reply:
> If you are using BEM, here is mistake. You can't use `nav__link` element if you don't have `nav` block

👀